packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -31,6 +31,8 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
             data-exclude-categories="$ctrl.excludeCategories"
             data-on-change="$ctrl.onFlavorCategorySelect(flavor, category)"
             data-catalog-endpoint="$ctrl.catalogEndpoint"
+            data-reload="$ctrl.reloadFlavorList"
+            data-load-end="$ctrl.onRegionChange()"
```

Comment:
As it is hard enough to understand the data flow, a cleaner solution would be to call a `onFlavorListLoadEnd()` method of the controller
```javascript
onFlavorListLoadEnd() {
  if (this.reloadFlavorList && this.currentStep === 2) {
    this.reloadFlavorList = false;
    this.onRegionChange();
  }
}
```
Something like this could explain more why we must call the `onRegionChange` method